DigitalLearningSolutions.Web/Views/RegisterDelegateByCentre/Summary.cshtml

```diff
@@ -134,6 +134,6 @@
     <form method="post" asp-action="Summary">
       <button class="nhsuk-button" type="submit">Save</button>
     </form>
-    <vc:back-link asp-controller="RegisterDelegateByCentre" asp-action="@Model.PreviousAction" link-text="Go back" />
+    <vc:cancel-link asp-controller="AllDelegates" asp-action="Index" />

```

Comment:
I think this was the only use of PreviousAction, could you remove it from the VM please if there are no other references to it.